src/webgpu/shader/execution/shader_io/fragment_builtins.spec.ts

```diff
@@ -1424,6 +1427,384 @@ g.test('inputs,sample_mask')
     );
   });
 
-g.test('subgroup_size').unimplemented();
+const kSizes = [
+  [15, 15],
+  [16, 16],
+  [17, 17],
+  [19, 13],
+  [13, 10],
+  [111, 2],
+  [2, 111],
+  [35, 2],
+  [2, 35],
+  [53, 13],
+  [13, 53],
+] as const;
+
+/**
+ * @returns The population count of input.
+ *
+ * @param input Treated as an unsigned 32-bit integer
+ */
+function popcount(input: number): number {
```

Comment:
For later:  I suggest refactoring this into a common spot, and calling it popcountU32 and having it assert the value is suitable for u32: in the right range, and integral.